spotbugs-exclude.xml

```diff
@@ -152,4 +152,13 @@
         <Class name="software.amazon.lambda.powertools.logging.internal.LambdaLoggingAspect"/>
         <Method name="setLogLevelBasedOnSamplingRate"/>
     </Match>
+    <Match>
```

Comment:
Spotbugs wasn't run on the examples previously. This lint comes from the original projects and I have not yet corrected it.